csrankings-x.csv

```diff
@@ -235,6 +235,7 @@ Xin Wang 0002,Fudan University,http://homepage.fudan.edu.cn/xinw2013,4KcxzhoAAAA
 Xin Wang 0019,Tsinghua University,http://mn.cs.tsinghua.edu.cn/xinwang,YPOBHYUAAAAJ
 Xin Wang 0061,Univ. of California - Santa Cruz,https://eric-xw.github.io,YjqluE0AAAAJ
 Xin Xia 0001,Monash University,https://xin-xia.github.io,XSZRxOEAAAAJ
+Xin Xin,Shandong University,https://xinxin-me.github.io,cYvuOUMAAAAJ

```

Comment:
This is a disambiguation entry; please add the correct suffix. See https://dblp.org/pid/35/1895.html